release_cloudbuild.yaml

```diff
@@ -13,7 +13,7 @@ steps:
     args: ['build', '//:go_default_test']
 # Upload the linux binary to GCS
   - name: 'gcr.io/cloud-builders/gsutil'
-    args: ['cp', 'bazel-bin/linux_amd64_stripped/go_default_test', 'gs://container-structure-test/$TAG_NAME/container-structure-test']
```

Comment:
FYI this one-liner seems to work if we don't want to have to keep battling with this:
```shell
bazel info | grep bazel-bin | cut -d' ' -f2
```

Reply:
works for me, this is becoming a nuisance

Reply:
going to fix this in a follow up PR: https://github.com/GoogleCloudPlatform/container-structure-test/issues/66